macaroons/README.md

```diff
@@ -114,6 +114,22 @@ The macaroon created by this call would only be allowed to call the `GetInfo` an
 `GetVersion` methods instead of all methods that have similar permissions (like
 `info:read` for example).
 
+Might you need a macaroon file with rights similar to admin.macaroon for custom 
+use case you can create such like in this example:
+
+```shell
+$  lncli bakemacaroon --save_to lnbits.macaroon \
+   address:read address:write \
+   info:read info:write \
+   invoices:read invoices:write \
+   macaroon:generate macaroon:read macaroon:write \
```

Comment:
Might make sense to add a sentence that when `macaroon:generate` is granted, then that macaroon can be used to create **new** macaroons with more permissions than the original macaroon. So this permission should be used with caution.

Reply:
Added.